aws.dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM maven:3-jdk-8-alpine as java-build
+
+COPY kinesis-client /opt/sites/rig/kinesis-client
+
+WORKDIR /opt/sites/rig/kinesis-client
+
+# Compile AWS Kinesis Java application
+RUN mvn package
+
+FROM elixir:1.6-alpine as elixir-build
+
+# Install Elixir & Erlang environment dependencies
+RUN mix local.hex --force
+RUN mix local.rebar --force
+RUN apk update && apk add make \
+    gcc \
+    g++
+
+ENV MIX_ENV=prod
+
+WORKDIR /opt/sites/rig
+
+# Copy release config
+COPY version /opt/sites/rig/
+COPY rel /opt/sites/rig/rel/
+COPY vm.args /opt/sites/rig/
+
+# Copy necessary files for dependencies
+COPY mix.exs /opt/sites/rig/
+COPY mix.lock /opt/sites/rig/
+COPY apps/rig/mix.exs /opt/sites/rig/apps/rig/
+COPY apps/rig_api/mix.exs /opt/sites/rig/apps/rig_api/
+COPY apps/rig_auth/mix.exs /opt/sites/rig/apps/rig_auth/
+COPY apps/rig_inbound_gateway/mix.exs /opt/sites/rig/apps/rig_inbound_gateway/
+COPY apps/rig_outbound_gateway/mix.exs /opt/sites/rig/apps/rig_outbound_gateway/
+
+# Install project dependencies
+RUN mix deps.get
+
+# Copy application files
+COPY config /opt/sites/rig/config
+
+COPY apps/rig/config /opt/sites/rig/apps/rig/config
+COPY apps/rig/lib /opt/sites/rig/apps/rig/lib
+
+COPY apps/rig_api/config /opt/sites/rig/apps/rig_api/config
+COPY apps/rig_api/lib /opt/sites/rig/apps/rig_api/lib
+COPY apps/rig_api/priv /opt/sites/rig/apps/rig_api/priv
+
+COPY apps/rig_auth/config /opt/sites/rig/apps/rig_auth/config
+COPY apps/rig_auth/lib /opt/sites/rig/apps/rig_auth/lib
+
+COPY apps/rig_inbound_gateway/config /opt/sites/rig/apps/rig_inbound_gateway/config
+COPY apps/rig_inbound_gateway/lib /opt/sites/rig/apps/rig_inbound_gateway/lib
+COPY apps/rig_inbound_gateway/priv /opt/sites/rig/apps/rig_inbound_gateway/priv
+
+COPY apps/rig_outbound_gateway/config /opt/sites/rig/apps/rig_outbound_gateway/config
+COPY apps/rig_outbound_gateway/lib /opt/sites/rig/apps/rig_outbound_gateway/lib
+
+# Compile and release application production code
+RUN mix release
+
+FROM erlang:20-alpine
+
+RUN apk update && apk add bash
+
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV REPLACE_OS_VARS=true
+
+# Install Java
+RUN { \
+		echo '#!/bin/sh'; \
+		echo 'set -e'; \
+		echo; \
+		echo 'dirname "$(dirname "$(readlink -f "$(which javac || which java)")")"'; \
+	} > /usr/local/bin/docker-java-home \
+	&& chmod +x /usr/local/bin/docker-java-home
```

Comment:
why is [`apk add openjdk8-jre`](https://pkgs.alpinelinux.org/package/edge/community/x86/openjdk8-jre) not enough?

Reply:
based on official Java Dockerfile https://github.com/docker-library/openjdk/blob/9a0822673dffd3e5ba66f18a8547aa60faed6d08/8-jdk/alpine/Dockerfile might try if this will suffice